src/screens/Likes.jsx

```diff
@@ -1,19 +1,23 @@
 import React, { useEffect } from "react";
 import styles from "./Likes.module.css";
 
-import { fetchLikes } from "../utils/request";
+import { fetchLikes, getMatches } from "../utils/request";
 import { useState } from "react";
 import Loader from "../components/Loader";
 import Swiper from "../components/Swiper";
 import Match from "../components/Match";
 
 const Likes = () => {
   const [likes, setLikes] = useState(null);
+  const [match, isMatch] = useState(null);
 
   useEffect(() => {
     const onMount = async () => {
       const likes = await fetchLikes();
+      console.log(likes);
       setLikes(likes);
+      const matched = await getMatches();
```

Comment:
I think this can get buggy. The `matches/new` returns all the matches that a candidate had. With the existing code, it worked because I suspect you had only 1 match. But if you have more than one, line 36 match.lab I think would fail. 

I solved it a bit different. In the Swiper component, when I `like` a user, I check to see if the like returns a `match`. 
If it returns a `match` I display the `Match` component.

I think its my fault because I didn't explain properly what all endpoints do.